doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/3_5_0.xhtml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<link rel="stylesheet" type="text/css" href="new.css" />
+<title>What's new in hale studio 3.5?</title>
+</head>
+<body>
+	<h1>What's new in hale studio 3.5?</h1>
+
+	<ul>
+		<li>Deegree workspace configuration export</li>
+		<li>GML partitioning by spatial extent</li>
+		<li>Support for plugin installation via update sites</li>
+	</ul>
+
+	<p>
+		For a full list of changes, please check out the <a
+			href="https://github.com/halestudio/hale/blob/master/CHANGELOG.md#350"
+			target="_blank">changelog on GitHub</a>
+	</p>
+
+	<p>The majority of the new features and improvements were made
+	   possible through our contracts with more than 30 customers. We'd like
+	   to thank you all for your support!</p>
+
+	<table border="0" cellpadding="15" cellspacing="0" class="news" id="3_5_0">
+		<tbody>
+			
+			<!-- New features -->
+			<tr>
+				<td class="section" colspan="2" id="3_5_0_Features" rowspan="1"><h2>New features</h2></td>
+			</tr>
+
+			<tr id="deegree-config-export">
+				<td class="title">Deegree workspace configuration export</td>
+				<td class="content">
+					<p>
+						A bit like the Geoserver App-Schema and Xtraserver configuration, it is now also 
+						possible to generate deegree workspace configurations from hale studio alignments.
+						This feature is only available through the command line interface.
```

Comment:
No, it is actually only available via the UI right now.

Reply:
@thorsten-reitz Needs to be changed in the blog post as well

Reply:
I'd suggest to put `This feature is currently not available in the command line interface.` instead.

Reply:
Sounds good :+1: 
I think if the dependency would be added to hale-cli it could probably be used via the `schema rewrite` command, but I did not test that.

Reply:
I pushed new commits with updates. I also adapted the corresponding changelog entry.